libbeat/common/terminal/terminal.go

```diff
@@ -0,0 +1,57 @@
+package terminal
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+// ReadInput Capture user input for a question
+func ReadInput() (string, error) {
```

Comment:
Wonder if we want a 'secure' Input. One that doesn't echo.

Reply:
Yes its provided by "golang.org/x/crypto/ssh/terminal" which take care of the different implementation